src/user-federation/ldap/LdapSettingsConnection.tsx

```diff
@@ -40,6 +40,19 @@ const testLdapProperties: Array<keyof TestLdapConnectionRepresentation> = [
   "authType",
 ];
 
+type TestTypes = "testConnection" | "testAuthentication";
+
+const convertFormToSettings = (form: UseFormMethods) => {
+  const settings: TestLdapConnectionRepresentation = {};
+
+  testLdapProperties.forEach((key) => {
+    const value = _.get(form.getValues(), `config.${key}`);
+    settings[key] = _.isArray(value) ? value[0] : "";
```

Comment:
You can also use [`Array.isArray()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) here instead of lodash.
```suggestion
    settings[key] = Array.isArray(value) ? value[0] : "";
```